src/managers/input/InputManager.cpp

```diff
@@ -1184,13 +1167,20 @@ void CInputManager::setTouchDeviceConfigs() {
 
 void CInputManager::setTabletConfigs() {
     for (auto& t : m_lTablets) {
+        Debug::log(LOG, "Checking for config for device %s", t.name);
         const auto HASCONFIG = g_pConfigManager->deviceConfigExists(t.name);
 
-        if (HASCONFIG) {
-            const auto OUTPUT   = g_pConfigManager->getDeviceString(t.name, "output");
-            const auto PMONITOR = g_pCompositor->getMonitorFromString(OUTPUT);
+        if (wlr_input_device_is_libinput(t.wlrDevice)) {
+            Debug::log(LOG, "Device %s is a libinput device", t.name);
```

Comment:
useless looooog

Reply:
ah yes, left in in because I was surprised it didn't work at first, but now removed